Sources/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -626,7 +626,6 @@ private extension PurchasesOrchestrator {
                         self.storeKitWrapper.finishTransaction(transaction)
                     }
                 } else {
-                    Logger.error(error)
```

Comment:
This is already logged from:
```swift
let purchasesError = error.asPurchasesError
```

Reply:
I've been meaning to look into this- thank you!